### DISCUSSION TITLE
Share your user defined themes.

### DISCUSSION THREAD

**Discussion by krstp** [zed-industries/zed#7337] [Show and tell]
Loving the user defined themes feature. It really did change the quality of development. Finally I feel almost at home. Show off your themes, looking for inspiration on different definitions.

![ZedThemeKrstp](https://github.com/zed-industries/zed/assets/4711498/b3727f4c-7f51-4e00-a6dc-a3b8413bf0ca)

**Comment by epmoyer** [comment]
# Monokai
I've published a set of 3 Monoakai Zed themes at [https://github.com/epmoyer/Zed-Monokai-Theme](https://github.com/epmoyer/Zed-Monokai-Theme)

<img width="501" alt="screenshot_monokai" src="https://github.com/zed-industries/zed/assets/136718/f2ff8dcf-ae48-4ce2-9954-423c077a0807">

**Comment by d1y** [comment]
<img width="1273" alt="image" src="https://github.com/zed-industries/zed/assets/45585937/9512bcbb-cc9f-4700-9d2c-e579780e02ca">

**Reply by d1y** [reply]
<img width="1515" alt="image" src="https://github.com/zed-industries/zed/assets/45585937/ac5f531d-7f73-425f-9b69-563f1288031d">

**Reply by etherealblaade** [reply]
Is there a chance to see any dots for this one?

**Reply by d1y** [reply]
zed-preview install this
<img width="766" alt="image" src="https://github.com/zed-industries/zed/assets/45585937/28cc60f8-d320-46c7-804d-aefa8d87cb2c">

**Reply by p6laris** [reply]
I really like the terminal, what is this theme? If it's possible can I know what are the configs

**Reply by alexandreruban** [reply]
@d1y your catppuccin-unofficial theme looks amazing 😍
Did you share the `.json` file somewhere by any chance?

**Comment by alygin** [comment]
Nord.

<img width="1331" alt="zed-nord" src="https://github.com/zed-industries/zed/assets/2101250/b73bf391-c88f-410b-bd3a-b832cce55f3e">

**Reply by diocletiann** [reply]
Theme looks great, hope you can share. How did you disable the tab bar?

**Reply by alygin** [reply]
> hope you can share

Sure! Here it is: [nord-theme.json](https://github.com/zed-industries/zed/files/14170880/nord-theme.json).

> How did you disable the tab bar?

This is a build with changes from #7356. Unfortunately, it hasn't been merged yet. We need to do some preliminary work to make tabless editing more friendly.

**Reply by alygin** [reply]
With several minor fixes: [nord-theme.json](https://github.com/zed-industries/zed/files/14230015/nord-theme.json)

**Comment by tecandrew** [comment]
# Catppuccin Mocha

![image](https://github.com/zed-industries/zed/assets/21319730/36d36c24-0d58-498d-bb1f-281a3d270e87)

**Reply by mauscoelho** [reply]
Could you share your theme?

**Reply by tecandrew** [reply]
sure! https://github.com/catppuccin/zed

**Reply by karimlevallois** [reply]
Well this is exciting, my favourite scheme!

**Reply by GonzaloSantos1** [reply]
What's your buffer font? I really like it!

**Reply by tecandrew** [reply]
@GonzaloSantos1 iirc, this was either [IBM Plex Mono](https://github.com/IBM/plex) or [Geist Mono](https://github.com/vercel/geist-font). i switch often, usually between these or [MonoLisa](https://www.monolisa.dev/) 😄

**Reply by XaviCode1000** [reply]
I recommend trying "MONASPACE" with "A".

https://github.com/githubnext/monaspace

**Comment by mauscoelho** [comment]
Helix default theme
https://github.com/mauscoelho/zed-helix-theme

![image](https://github.com/zed-industries/zed/assets/5712300/3a77687f-ffd4-4746-ad9a-243e606afcae)

**Comment by rubjo** [comment]
Ultimate Dark Neo, a port of a Sublime Text / VS Code theme
https://github.com/rubjo/ultimate-dark-neo-zed

<img width="1451" alt="SCR-20240208-llre" src="https://github.com/zed-industries/zed/assets/42270947/aac8736b-22a7-4a56-9206-fddd92948373">

**Reply by Nishantdd** [reply]
Hey mate, can you share your font_features? I just can't get it to work and I'm pretty sure I'm doing it right:
```javascript
"buffer_font_features": {
    "calt": true,
    "ss01": true
  },
```
  But it doesn't affect anything

**Reply by rubjo** [reply]
Sorry, didn't see this comment until now. I have no `buffer_font_features` in my settings...

Using your setting does have an effect, though.

**Comment by brunowilliang** [comment]
🚀 New Theme Announcement: ZedSpace
https://github.com/Brunowilliang/zedspace

![preview](https://github.com/zed-industries/zed/assets/41432581/56cd645b-3cd6-4472-ac93-8088d8ad6e35)

Enjoy 🥳

**Comment by carlocaione** [comment]
### NeoSolarized Zed theme 
https://github.com/carlocaione/NeoSolarized.zed
<img width="2032" alt="Screenshot 2024-02-09 at 15 32 54" src="https://github.com/zed-industries/zed/assets/357758/6e3777e1-595c-4030-ac1c-2bbc73acb069">
<img width="2032" alt="Screenshot 2024-02-09 at 15 34 01" src="https://github.com/zed-industries/zed/assets/357758/61b3bb24-556f-4d12-8696-3477a8b6271f">

**Comment by Otterpocket** [comment]
Dracula 🧛

https://github.com/dracula/zed

<img width="1552" alt="image" src="https://github.com/zed-industries/zed/assets/7274458/9f0f33a8-2280-49f9-8a1e-af5b670ba3dd">

**Comment by JosephTLyons** [comment]
Here's my obligatory comment, encouraging everyone who has made a theme to open a PR on our extension store repository!

https://github.com/zed-industries/extensions

**Comment by xmonkee** [comment]
[Visual Studio Light and Dark themes](https://gist.github.com/xmonkee/99f386928131e6acd588033b29c31dea)

<img width="1822" alt="image" src="https://github.com/zed-industries/zed/assets/5104338/fff934b7-3614-4369-bdd1-f1c42845d350">
<img width="1822" alt="image" src="https://github.com/zed-industries/zed/assets/5104338/05d6b2fb-e157-46cd-a6d7-60b25e389f2a">

**Comment by kdubrovsky** [comment]
I've just created my own theme [Kiselevka](https://github.com/kdubrovsky/kiselevka)

Now you can install it from Zed extensions. 

![image](https://github.com/zed-industries/zed/assets/109236274/81736387-bd52-4e03-823b-1c475b8d56ef)

**Comment by zarifpour** [comment]
<div align="center">
<img alt="XY-Zed logo" width="" height="100" src="https://github.com/zed-industries/zed/assets/16494335/707c245f-5382-452e-a427-338f689fb0b8">
</div>

![example](https://github.com/zed-industries/zed/assets/16494335/b41f31a3-44d5-4e42-b278-f465c19ec43e)

**Reply by LittleFat3Tuan** [reply]
Fancy colors and dark enough background. Absolutely love it.

**Comment by etiennelacoursiere** [comment]
I always liked the colors i get when i open Rose Pine in tmux without the option `screen-256color` set. 
So i tried to recreate that theme for zed.

Broken Pine: 

![image](https://github.com/zed-industries/zed/assets/12648647/24204b1e-2b69-4349-bc61-dea68ac5e9da)
![image](https://github.com/zed-industries/zed/assets/12648647/8fd7d4f8-0552-4eb7-9abb-4466267ae78a)

**Reply by mkweick** [reply]
This is S tier! Do you have a link to these theme settings?

**Reply by etiennelacoursiere** [reply]
Sure! There you go: https://github.com/etiennelacoursiere/zed-themes

**Comment by Imgkl** [comment]
Hope it's not too late to showcase the theme. [The Dark Side.](https://github.com/Imgkl/the-dark-side) 

<img width="1800" alt="image" src="https://github.com/zed-industries/zed/assets/25414681/a982cea4-106b-43ae-8b01-58164a9294f6">

<img src="https://i.ibb.co/f2SLdm4/Screenshot-2024-02-25-at-11-29-53-AM.png"/>

**Comment by erichjrusch** [comment]
Zed Cyberpunk - Based off of the popular [VSCode Cyberpunk.](https://marketplace.visualstudio.com/items?itemName=max-SS.cyberpunk)

It's not mine to share, but Zed now truly feels like home. 🥰

<img width="1840" alt="Screenshot 2024-03-15 at 1 37 52 PM" src="https://github.com/zed-industries/zed/assets/94961/7b40d58a-abde-4991-a7c8-3bec84937da9">

**Reply by notsoharshdevisha** [reply]
😍 by any chance could you link this theme please?

**Comment by babyccino** [comment]
Borderless Ayu https://github.com/babyccino/Borderless-Ayu-Zed. Just matches the Ayu themes in VSCode

![](https://raw.githubusercontent.com/babyccino/Borderless-Ayu-Zed/main/img/mirage.png)

**Comment by borngraced** [comment]
My personal Yin inspired theme for less distractions :))))
https://github.com/borngraced/Monosami?tab=readme-ov-file
<img width="1767" alt="sample" src="https://github.com/zed-industries/zed/assets/51881311/ebf34f7e-2528-4087-958f-4e1d5ae4ab88">

**Comment by felipetesc** [comment]
Inspired on Vapor Theme from SteamOS KDE Plasma 6:
https://github.com/felipetesc/zed-vapor/
![zed_vapor](https://github.com/zed-industries/zed/assets/2336812/fc5bafbb-cfc7-4543-a45d-1180417fc153)

**Comment by denix666** [comment]
https://github.com/denix666/zed-theme

![screenshot](https://github.com/zed-industries/zed/assets/1541240/59ffb2eb-b9ee-4c37-8736-8afbac7998f7)

**Comment by davccavalcante** [comment]
🟢 Brings the nostalgic essence of vintage green phosphor monitors to your Zed editor. This theme is crafted to offer a unique and immersive visual experience, capturing the original charm of early computing displays. Available in both dark and light versions, the Green Monochrome Monitor Theme transforms your coding environment with its distinct green-on-black and green-on-light-green aesthetics.

https://github.com/Takk8IS/green-monochrome-monitor-theme-for-zed

Green Monochrome Monitor Dark Theme
![screenshot-dark](https://github.com/zed-industries/zed/assets/152639968/fc8aecda-380a-4a27-8318-79c653304ad1)

Green Monochrome Monitor Light Theme
![screenshot-light](https://github.com/zed-industries/zed/assets/152639968/bf375857-4f65-4834-ae67-8f62ad479a42)

Experience the originality and unique aesthetics of the Green Monochrome Monitor Theme and take your coding journey back to the era of early computing displays.

**Comment by meocoder31099** [comment]
For those who like a relaxed, gentle, and modern style, I have ported the Serendipity theme for Zed
[Serendipity-Theme](https://github.com/meocoder31099/Serendipity-Theme-Zed)
### Options
You have three themes, Sunset, Midnight and Morning and here the screenshots

### Midnight
![Midnight Preview](https://raw.githubusercontent.com/meocoder31099/Serendipity-Theme-Zed/main/assets/previews/midnight.png?raw=true)

### Midnight minimal
![Midnight Preview](https://raw.githubusercontent.com/meocoder31099/Serendipity-Theme-Zed/main/assets/previews/midnight-minimal.png?raw=true)

### Sunset
![Sunset Preview](https://raw.githubusercontent.com/meocoder31099/Serendipity-Theme-Zed/main/assets/previews/sunset.png?raw=true)

### Morning
![Morning Preview](https://raw.githubusercontent.com/meocoder31099/Serendipity-Theme-Zed/main/assets/previews/morning.png?raw=true)

**Comment by ivryb** [comment]
One Dark Ocean, with darker background for better contrast.

![screenshot 2024-06-06 at 21 55 28@2x](https://github.com/zed-industries/zed/assets/13164347/2a958ab4-0b6c-4ce9-befe-ab9d9989795f)

**Reply by ivryb** [reply]
https://gist.github.com/ivryb/4b50dfc2349fb20209039370a208ff7f

**Comment by xambassador** [comment]
[Sinatra Theme](https://github.com/xambassador/sinatra-zed-theme), feel the night.

## Sinatra Dark Borderless

![ss-1](https://github.com/zed-industries/zed/assets/69746573/188586fd-6642-4f28-8447-f728d55080a4)

## Sinatra Dark

![ss-2](https://github.com/zed-industries/zed/assets/69746573/a7b854ec-7d59-4771-b31d-cefed66021ca)

**Reply by Otterpocket** [reply]
This looks really nice

**Comment by Moosems** [comment]
Late Night
<img width="1552" alt="screenshot_late_night" src="https://github.com/zed-industries/zed/assets/95927277/a63645db-b7b5-415f-b81e-e8a3855a3f72">

**Comment by kirqe** [comment]
https://github.com/kirqe/zed-ocean-dark-motifs-theme

<img width="988" alt="theme" src="https://github.com/zed-industries/zed/assets/5272402/25ad8a1a-c9f1-44ad-a0c5-19ec996aa40d">

**Comment by JewinV** [comment]
<h2>Dark Purple Monokai</h2>

![python](https://github.com/zed-industries/zed/assets/86153268/0df3e91f-2be1-4750-9a1d-b68c70e5607e)

https://github.com/JewinV/dark-purple-monokai-zed-theme

**Comment by kirademiurge** [comment]


**Reply by Moosems** [reply]
Comments?

**Comment by Demonstrandum** [comment]
I ported the [Iceberg](https://cocopon.github.io/iceberg.vim/) themes (from (Neo)Vim)

**Source:** [gist.github.com/Demonstrandum/487f1939915d047f945f094b75104c15](https://gist.github.com/Demonstrandum/487f1939915d047f945f094b75104c15)

<img width="1402" alt="Screenshot 2024-07-10 at 20 07 14" src="https://github.com/zed-industries/zed/assets/26842759/df8c3b2d-7021-45a6-8957-a2802ac29c05">
<img width="1402" alt="Screenshot 2024-07-10 at 20 07 30" src="https://github.com/zed-industries/zed/assets/26842759/ef154b64-2fa1-47ff-8aa2-cddcbf4f38d6">

**Comment by fartoot** [comment]
## Material Ocean
Theme: https://gist.github.com/fartoot/6d84fbb1d35ddee858d386502305f31f

![ocean-2](https://github.com/zed-industries/zed/assets/17651064/6e0dc117-c2e7-4da8-a78a-066468751cb4)

**Comment by salihdev0** [comment]
One Dark Pro Monokai Darker

Repo: https://github.com/9ssi7/zed-one-dark-pro-monokai-darker

<img width="1362" alt="Screenshot 2024-07-11 at 21 35 45" src="https://github.com/user-attachments/assets/962249ca-b895-49f2-b530-b955b0b8a6af">
<img width="1366" alt="Screenshot 2024-07-11 at 21 36 34" src="https://github.com/user-attachments/assets/02f2d8a2-3a17-4d03-bbdc-034a193f9b6e">

**Comment by ProDeSquare** [comment]
**Marine Dark** theme

Repo: [MarineDark/marine-dark.zed](https://github.com/MarineDark/marine-dark.zed)

![Marine Dark Screenshot](https://github.com/user-attachments/assets/4e55b0f1-120a-4230-a101-ac0f8bb21836)
*Zed running on Arch GNU/Linux*

**Comment by sidharthpunathil** [comment]
## macOS Dark Theme Enhanced
Repo: https://github.com/sidharthpunathil/zed-theme-macos-dark-enhanced
---
![Screenshot from 2024-07-18 18-23-50](https://github.com/user-attachments/assets/de68e8da-2181-408a-bdc0-fe100077ac9a)

**Comment by tty2** [comment]
[Monkey theme](https://github.com/tty2/monkey-zed-theme)
Monokai inspired with dark blue backgrounds and more contrast

![screen](https://github.com/user-attachments/assets/bd2e089e-0517-4d84-a6d5-da551d44a7ba)

**Comment by tokiory** [comment]
Gruvbox Material, you can grab it here: https://github.com/tokiory/zed-gruvbox-material

<img width="1624" alt="image" src="https://github.com/user-attachments/assets/dd142fe7-26d8-4c5c-8011-26f44321d02e">

**Comment by ncpa0cpl** [comment]
[GnomeDarkThemeZed](https://github.com/ncpa0cpl/GnomeDarkThemeZed/blob/master/README.md)

Gnome Dark theme based on [GNOME theme for VSCode](https://marketplace.visualstudio.com/items?itemName=rafaelmardojai.vscode-gnome-theme)

![gnome-dark](https://github.com/user-attachments/assets/ba6f7929-4cd2-4f71-a3bb-ff983f1f2d37)

**Comment by vjdw** [comment]
I ran the VS Code Night Owl themes through the importer and got [this](https://gist.github.com/vjdw/8bf211f36eab828e077eed9477e44daf).

![image](https://github.com/user-attachments/assets/aba16243-f337-4024-8c75-8813db490f74)

![image](https://github.com/user-attachments/assets/d3c86694-daa3-4b64-a9e5-1211fa01cabe)

**Reply by XaviCode1000** [reply]
Can the colors and icons of the file folders on the panel be customized?

**Comment by JoaoGaValentim** [comment]
# Zed Omni
My fork of the Darcula Theme [Zed Omni Theme](https://github.com/JoaoGaValentim/zed_omni/)
![zed_omni](https://github.com/JoaoGaValentim/zed_omni/raw/main/.github/assets/img/zed_omni.png)

**Comment by JoaoGaValentim** [comment]
# Pickle Rick Theme's
Rick & Morty Theme! [Get theme](https://github.com/JoaoGaValentim/picklerick_zed)
![picklerick_zed](https://github.com/user-attachments/assets/7fb423ef-0ae7-4ff0-8eb2-7fae40748507)

**Comment by Soupertonic** [comment]
# Mur Mur
<img width="1798" alt="Screenshot 2025-05-27 at 03 07 49" src="https://github.com/user-attachments/assets/27de737d-a868-4cf8-a5a8-e5561d740846" />

**Reply by soymrunkn0wn** [reply]
no repo?

**Comment by meocoder31099** [comment]
# [GitHub Classic](https://github.com/meocoder31099/Github-Classic-Theme-Zed)

![github-dark-classic](https://github.com/user-attachments/assets/a1b199d3-310d-4c5a-84ba-15b9c34a6373)

![github-light-classic](https://github.com/user-attachments/assets/6b86fa45-197b-43e1-98b2-b131e9f5a273)

**Comment by necm1** [comment]
https://github.com/necm1/v-zed-theme

![screenshot](https://github.com/user-attachments/assets/9e70beaf-5a33-4fbb-b798-eb3cd039f2aa)

**Comment by BlackGhost10** [comment]
<img width="1920" height="1048" alt="image" src="https://github.com/user-attachments/assets/acabb54a-1121-4135-a0ee-fd8261f598ce" />
https://zed-themes.com/themes/6ONKpBlRuTcP7bHUK138b?name=BlackJ here my custom theme enjoy it :)
https://github.com/BlackGhost10/zed-BlackJ-theme
<img width="1503" height="872" alt="ScreenShot" src="https://github.com/user-attachments/assets/3bbabcc8-473e-44e1-8e09-1123e8c1a61a" />

**Comment by somepaulo** [comment]
[Adwaita](https://github.com/somepaulo/adwaita-zed-theme)

<img width="1488" height="906" alt="Screenshot From 2025-07-15 23-27-41" src="https://github.com/user-attachments/assets/39350a2e-246c-4adb-9091-f012569b50ab" />
<img width="1488" height="906" alt="Screenshot From 2025-07-15 23-27-33" src="https://github.com/user-attachments/assets/2d8554b0-2247-4337-805e-ef01ff82e150" />

**Comment by merrynuts** [comment]
[merrynuts](https://github.com/merrynuts/zeditor-theme)
<img width="1917" height="1033" alt="screenshot" src="https://github.com/user-attachments/assets/cc2a0489-97ae-47b5-ad95-dfa77f34851c" />

**Comment by fonnesbeck** [comment]
# Cobalt2 

A port of the popular [Cobalt2](https://github.com/wesbos/cobalt2-vscode) theme by [Wes Bos](https://wesbos.com/)

https://github.com/fonnesbeck/cobalt2-zed

<img width="1938" height="1534" alt="image" src="https://github.com/user-attachments/assets/1c614d15-13d9-4bdd-9be2-78e43e5e5fd5" />

**Comment by azzamsa** [comment]
Modus Vivendi Tinted + Catppuccin Mocha

```json
  "theme_overrides": {
    "Modus Vivendi Tinted": {
      // Tab
      "tab_bar.background": "#000000",
      "tab.inactive_background": "#11111b",
      "tab.active_background": "#1e1e2e",
      // Editor
      "editor.background": "#000000",
      "editor.gutter.background": "#11111b",
      "syntax": {
        "comment": {
          "color": "#a6adc8"
        }
      },
      "players": [
        {
          "selection": "#313244"
        }
      ]
    }
  }
```

<img width="1920" height="1042" alt="image" src="https://github.com/user-attachments/assets/36835734-2bee-446e-a9bb-7c97b540ec9b" />

**Comment by o-gi** [comment]
Can't imagine life without Zed anymore

https://github.com/o-gi/gruber-darker-rin.zed

<img width="1919" height="1053" alt="screenshot-blur" src="https://github.com/user-attachments/assets/66620026-962d-41b2-a7f4-779fe0570c82" />

**Comment by garland-culbreth** [comment]
[Ayu Mirage Bordered](https://github.com/garland-culbreth/ayu-mirage-bordered-zed)

Didn't love the default Ayu Mirage so made one closer to [vscode-ayu's](https://github.com/ayu-theme/vscode-ayu) Ayu Mirage Bordered. Only did the Mirage colorway for the moment.

<img width="969" height="703" alt="image" src="https://github.com/user-attachments/assets/c21bada0-6995-4b61-854c-9851bc51fae5" />

**Comment by davidas13** [comment]
Kanagawa Monochrome

https://codeberg.org/davidas13/cet-kanagawa-monochrome.zed

![preview](https://codeberg.org/davidas13/cet-kanagawa-monochrome.zed/media/branch/main/docs/assets/preview.webp)

**Comment by gv-sh** [comment]
Here's Ink Signal, a theme pack I made. Five painting-inspired palettes, each light and dark, with fewer syntax colours.

The pack is paid; you can try the colours free in the preview: https://gaurav-singh.info/ink-signal/

<img width="1333" height="768" alt="water-lilies-morning" src="https://github.com/user-attachments/assets/571d8fcc-f0ad-4119-a90d-fc8cf7442031" />
